lib/basedriver/capabilities.js

```diff
@@ -25,7 +26,7 @@ function validateCaps (caps, constraints = {}, opts = {}) {
   let  {skipPresenceConstraint} = opts;
 
   if (!_.isObject(caps)) {
```

Comment:
should it be isPlainObject instead?

Reply:
I may as well change it to `isPlainObject` although because it's parsed from JSON it couldn't be a non-plain object.